book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :author, :title
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rentals(rent)
+    @rentals.push(rent)
+  end
```

Comment:
- [ ] In The add_rentals method, you have to create a rental passing all the required parameters. Here, the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association
```ruby
def add_rental(person, date)
  Rental.new(date, person, self)
end
```